src/library/scala/util/Either.scala

```diff
@@ -425,62 +405,58 @@ object Either {
     }
   }
 
-  /**
-   * Projects an `Either` into a `Left`.
+  /** Projects an `Either` into a `Left`.
    *
-   * This allows for-comprehensions over the left side of Either instances,
-   * reversing Either's usual right-bias.
+   *  This allows for-comprehensions over the left side of Either instances,
+   *  reversing Either's usual right-bias.
    *
-   * For example {{{
-   * for (s <- Left("flower").left) yield s.length // Left(6)
-   * }}}
-   *
-   * Continuing the analogy with [[scala.Option]], a `LeftProjection` declares
-   * that `Left` should be analogous to `Some` in some code.
+   *  For example {{{
+   *  for (s <- Left("flower").left) yield s.length // Left(6)
+   *  }}}
    *
-   * {{{
-   * // using Option:
-   * def interactWithDB(x: Query): Option[Result] =
-   *   try {
-   *     Some(getResultFromDatabase(x))
-   *   } catch {
-   *     case ex => None
-   *   }
-   *
-   * // this will only be executed if interactWithDB returns a Some
-   * val report =
-   *   for (r <- interactWithDB(someQuery)) yield generateReport(r)
-   * if (report.isDefined)
-   *   send(report)
-   * else
-   *   log("report not generated, not sure why...")
-   * }}}
+   *  Continuing the analogy with [[scala.Option]], a `LeftProjection` declares
+   *  that `Left` should be analogous to `Some` in some code.
    *
-   * {{{
-   * // using Either
-   * def interactWithDB(x: Query): Either[Exception, Result] =
-   *   try {
-   *     Right(getResultFromDatabase(x))
-   *   } catch {
-   *     case ex => Left(ex)
-   *   }
-   *
-   * // this will only be executed if interactWithDB returns a Right
-   * val report =
-   *   for (r <- interactWithDB(someQuery).right) yield generateReport(r)
-   * if (report.isRight)
-   *   send(report)
-   * else
-   *   log("report not generated, reason was " + report.left.get)
-   * }}}
+   *  {{{
+   *  // using Option:
+   *  def interactWithDB(x: Query): Option[Result] =
+   *    try {
+   *      Some(getResultFromDatabase(x))
+   *    } catch {
+   *      case ex => None
+   *    }
+   *
+   *  // this will only be executed if interactWithDB returns a Some
+   *  val report = for (r <- interactWithDB(someQuery)) yield generateReport(r)
+   *  if (report.isDefined)
+   *    send(report)
+   *  else
+   *    log("report not generated, not sure why...")
+   *  }}}
    *
-   * @author <a href="mailto:research@workingmouse.com">Tony Morris</a>, Workingmouse
-   * @version 1.0, 11/10/2008
+   *  {{{
+   *  // using Either
+   *  def interactWithDB(x: Query): Either[Exception, Result] =
+   *    try {
+   *      Right(getResultFromDatabase(x))
+   *    } catch {
+   *      case ex => Left(ex)
+   *    }
+   *
+   *   // this will only be executed if interactWithDB returns a Right
+   *   val report = for (r <- interactWithDB(someQuery).right) yield generateReport(r)
+   *   if (report.isRight)
```

Comment:
we don't want to encourage this style of coding, I think, especially the `report.left.get` that follows. could this be replaced with a pattern match?